AppCenterAnalytics/AppCenterAnalytics/MSACAnalytics.h

```diff
@@ -197,6 +197,18 @@ NS_SWIFT_NAME(Analytics)
  */
 + (void)resume;
 
+/**
+ * Manual start of session creation process.
```

Comment:
The description is not correct. Should be something like `Start session manually.`.